server/controllers/notification.js

```diff
@@ -0,0 +1,58 @@
+const Notification = require("../models/Notifications");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /notifications/create
+// @desc create new notification
+// @access Public
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  const notification = new Notification(({ type, title, description, read, date, image, recipient } = req.body));
+
+  const newNotification = await notification.save();
+  res.status(200).json({
+    success: {
+      notification: newNotification,
+    },
+  });
+});
+
+// @route PATCH /notifications/read
+// @desc mark notification as read
+// @access Public
+exports.readNotification = asyncHandler(async (req, res) => {
+  const notification = await Notification.findById(req.params.id);
```

Comment:
Optional: Similar to the comment above, is a good practice to validate the route parameters before passing them into a query :)